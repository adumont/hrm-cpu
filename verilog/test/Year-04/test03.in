25 94
06 d1
07 44
52 40
15 a7
35 dc
55 36
28 32
11 7d
48 a6
06 79
14 34
10 12
52 5a
51 76
47 00
39 35
57 9c
27 77
21 3e
57 24
08 04
04 b8
47 64
25 95
37 cc
33 da
57 52
46 bd
14 d5
02 a6
21 2e
35 15
40 75
15 87
49 42
08 71
19 80
35 53
45 f5
13 85
41 dc
38 b6
27 3d
25 11
24 18
20 1f
31 21
