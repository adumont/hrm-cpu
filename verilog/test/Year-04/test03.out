d1
94
40
44
dc
a7
32
36
a6
7d
34
79
5a
12
00
76
9c
35
3e
77
04
24
64
b8
cc
95
52
da
d5
bd
2e
a6
75
15
42
87
80
71
f5
53
dc
85
3d
b6
18
11
21
1f
